docs/casa/manual/chapters/20_sliding_door_assembly.md

```diff
@@ -0,0 +1,91 @@
+# Sliding Door Assembly
+
+![](../img/20_1.png)
+
+---
+
+!!! info annotate "Components Required"
+    ```
+        2 x Extrusion I
+        1 x Extrusion J
+        2 x M5 x 10 BHCS
+    ```
+![](../img/20_s1.png)
+
+---
+
+Slide the panel into the extrusion grooves. It will fit loosely at this stage.
```

Comment:
I used 1mm foam tape around the edges on both sides, as well as a couple of strips on the printed parts to eliminate rattle. Not sure if that's an official recommendation, but seems like something that might be mentioned. I am not sure that my windows were BOM thickness though, so it may be bad advice. Something to think about?

Reply:
Mine has stripping on one edge but it may benefit from having it on both on reflection. It was a 'loose' fit on one side, perhaps a too tight fit on two though.